src/ensembl/src/shared/badged-button/BadgedButton.scss

```diff
@@ -0,0 +1,20 @@
+@import 'src/styles/common';
+
+.badgedButton {
+  position: relative;
+  display: inline-flex;
+}
+
+.badgeDefault {
+  position: absolute;
+  background-color: $ens-green;
+  height: 26px;
+  width: 26px;
```

Comment:
Scary. How many characters will fit into this badge? Does Andrea offer any suggestions about what the badge should look like if there are several characters here? Is it possible that some kind of a rounded square (which can safely grow in width) will be a safer shape for the badge than a circle?

Reply:
At the moment it fits in three characters (eg. 999).
If I set `min-width: 26px` and `border-radius: 13px` it seems to expand safely. 
I'll check with Andrea if we need to allow more than 3 characters and make an update.

Reply:
As discussed with Steve and Andrea, we will not be having more than 3 characters displayed inside the badge.